plugins/woocommerce/includes/class-wc-post-types.php

```diff
@@ -369,6 +369,63 @@ public static function register_post_types() {
 			)
 		);
 
+		register_post_type(
+			'product_form',
+			apply_filters(
```

Comment:
@retrofox should we feature flag this work for now?
As I am not sure how invasive this is or what areas it could affect, that we may not be aware about.

Reply:
makes sense 👍

Reply:
https://github.com/woocommerce/woocommerce/pull/48136